gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -122,7 +122,8 @@ class VeloxTestSettings extends BackendTestSettings {
       // We can enable the below test for spark 3.4 and higher versions.
       "Gluten - describe",
       // decimal failed ut.
-      "SPARK-22271: mean overflows and returns null for some decimal variables"
+      "SPARK-22271: mean overflows and returns null for some decimal variables",
+      "SPARK-34165: Add count_distinct to summary"
```

Comment:
Can we explain the reason to exclude this test with some comments?

Reply:
Add comments, thanks!

Reply:
What is the issue for this test? Is it runnable with fallback?

Reply:
https://github.com/oap-project/gluten/issues/1659

Reply:
Got it, thank you. Do you plan to take a look？Otherwise, I can check it in the next few days. Another confusion is, how does this test relate to the change in this PR?

Reply:
> Do you plan to take a look？

I'm not familiar with these code, hope you guys could take it.

> how does this test relate to the change in this PR?

Somewhere in this UT will use first aggregate function, so it will fallback before.

Reply:
Understood.